recipes/zlib-ng/all/conanfile.py

```diff
@@ -106,6 +108,12 @@ def package_info(self):
             self.cpp_info.libs = [f"z{suffix}"]
         if self.options.zlib_compat:
             self.cpp_info.defines.append("ZLIB_COMPAT")
+            #copied from zlib
```

Comment:
I liked it, but it's not supported by the upstream.

I opened a PR to the upstream asking about it: https://github.com/zlib-ng/zlib-ng/pull/1601

Let's see what is the answer.

Reply:
change is available in latest 2.1.6. Should I add this version in this PR?

Reply:
Yes, please.

Reply:
2.1.6 was added in other PR. I've rebased my branch on latest master.